packages/nc-gui-v2/composables/useGlobalState/index.ts

```diff
@@ -1,6 +1,7 @@
 import { breakpointsTailwind, usePreferredLanguages, useStorage } from '@vueuse/core'
```

Comment:
As a compromise we could change `useGlobalState` to check whether `const { $state } = useNuxtApp()` is already initialized and if it is, we just return that from `useGlobalState` instead of doing the whole "initialization" process - then you could keep using `const globalState = useGlobalState()` without any complains from me 😝

Reply:
@bcakmakoglu Let keep the way it is. I should had read the comments in `useGlobalState` 😅

Reply:
Taken the suggestion where we return if the state is set in `useGlobalState`